src/solvers/Solver.cpp

```diff
@@ -358,6 +444,29 @@ void LiveNurse::computeMinMaxDaysNoPenaltyConsDay(State* pCurrentState, int nbDa
 		+ std::min(nbDaysMin%lengthStintMax, minConsDaysWork());
 }
 
+// Print the contract type + preferences
+void LiveNurse::printContractAndPrefenrences(Scenario *pScenario) const {
```

Comment:
small typo in function name printContractAndPrefe**n**rences

Reply:
Done